config-specs/paper/paper-global.yml

```
@@ -424,6 +424,13 @@ unsupported-settings:
     description: >-
       Whether the server should allow players to overstack items in the
       grindstone
+  allow-unsafe-end-portal-teleportation:
+    default: 'false'
+    description: >-
+      This setting allows for exploits related to end portal teleportation to be
+      possible. This includes sand duplication, and other exploits. This
```

this reads weird, if you only listing one, maybe we should just write something like
```suggestion
      This setting allows for exploits related to end portal teleportation to be
      possible, for example sand duplication. This
```